### PR TITLE
Normalise title casing

### PR DESCRIPTION
**Pull request by NVolcz**
I noticed that the title casing wasn't consistent and I normalised it to sentence casing since it seemed to be the most used style. Could also normalise to [title case](https://en.wikipedia.org/wiki/Title_case) if that's preferred.

### CODE REVIEW

File: website/public/locales/en.json [hunk 1]
```diff
@@ -41,19 +41,19 @@
...
+    "titleMessage": "Decrypted message",
     "subtitleMessage": "This secret might not be viewable again, make sure to save it now!",
     "buttonCopy": "Copy"
   },
   "delete": {
     "buttonDelete": "Delete",
     "messageDeleted": "The secret is removed from the server!",
-    "dialogTitle": "Delete the Secret?",
-    "dialogMessage": "Are you sure you want to remove this Secret?",
+    "dialogTitle": "Delete the secret?",
```
Comment (jhaals) [inline website/public/locales/en.json:55]:
```suggestion
    "dialogTitle": "Delete secret?",
```

Does this read nicer?

File: website/public/locales/en.json [hunk 2]
```diff
@@ -66,24 +66,24 @@
...
   "expiration": {
     "legend": "The encrypted message will be deleted automatically after",
-    "optionOneHourLabel": "One Hour",
-    "optionOneDayLabel": "One Day",
-    "optionOneWeekLabel": "One Week"
+    "optionOneHourLabel": "One hour",
+    "optionOneDayLabel": "One day",
+    "optionOneWeekLabel": "One week"
   },
   "features": {
```
Comment (jhaals) [inline website/public/locales/en.json:73]: It might be cleaner to keep title casing on the feature titles.

**Review by jhaals — COMMENTED** [review]
Thanks for the PR! I agree that some of these should be changed, added some suggestions

File: website/public/locales/en.json (hunk 2 above)
Comment (NVolcz) [reply website/public/locales/en.json:73]: Title casing on the features and sentence casing everywhere else? It looks weird when casings are mixed too me. Like when a codebase doesn't follow a consistent style.

**Comment by jhaals** [comment]
I agree, let's go with this 👍

**Pull request merged**